### ISSUE TITLE
Add a "Connect to Server" button on the web UI.

### ISSUE THREAD

**Issue by Cooldude2606** [clusterio/clusterio#608]
tl;dr There should be a button which can be clicked which will launch factorio and connect you to the selected server.

## Why
There has been some confussion about if starting a server should also launch factorio. This can be easily explained but having a deadicated "connect" button will prevent this being asked again. Further more, it would just be a really nice feature to have to quickly join a server rather than needing to use the server menu or direct connect.

## What
There should be a "Connect" or "Connect to Server" next to the start button on instance pages and/or as the URL for "Public Address" on the instances list.

## How
Steam provides support for starting game through URL app protocols, this includes providing arguments to the game being started. This can be used to launch factorio and connect to a server with minimal input from the user.
eg, `steam://run/427520//--mp-connect=play.explosivegaming.nl` (`427520` is the app id for Factorio)

## Issues
If the user already has factorio open then nothing will happen. This can't be detected directly but it is possible to check if the user is currently on a cluster instance and prompt a server switch, but this adds more complexity for something already possible in game with the server switch plugin.
